### ISSUE TITLE
Feature Request: 'Cancel' button

### ISSUE THREAD

**Issue by Bujiraso** [phw/peek#666]
Hello,
Thanks for this excellent program. I get great use out of it.

## Use Case
Sometimes I'm leave Peek recording the screen while attempting to coax specific behaviour out of a program to capture it.
I'll leave Peek running for say 2 - 3m while trying to prompt the behavior but sometimes I am totally out of luck and the resultant video will be worthless.

## Current Design
At this time I hit 'Stop' and the 'Rendering' phase of Peek begins. This is input dependent and can therefore take a while if I had left it running for a while.
This 'Rendering' phase is terminated by a file operation pop-up for saving.

## Feature
I would like to optionally skip the 'Rendering' phase when I don't need the output saved.

I would like to request a small feature of a "Cancel" button to skip the 'Rendering' phase and let me try again without waiting.
I am ok with either a new button beside "Stop" in red or whatever design fits best.

**Comment by stuaxo** [comment]
"Stop" should probably become "Save", then having something like "Discard" or "Cancel" as well would make sense.
